integrations/integration_test.go

```diff
@@ -182,7 +182,6 @@ func prepareTestEnv(t testing.TB, skip ...int) func() {
 	deferFn := PrintCurrentTest(t, ourSkip)
 	assert.NoError(t, models.LoadFixtures())
 	assert.NoError(t, os.RemoveAll(setting.RepoRootPath))
-	assert.NoError(t, os.RemoveAll(models.LocalCopyPath()))
```

Comment:
It isn't that it's not needed but that there's a race because we're using it for pulls and the pull task. If you try to delete it between tests, occasionally you'll get an error due to a testpullrequest being run. 

The reality is that localcopypath isn't what it used to be - in Gogs this was a full clone and checkout of every repository which had to be locked updated, and unlocked for every modification. We now use it only for extremely temporary very shallow clones with shared object DBs to the real repositories - avoiding checking out anything except in merge. It doesn't need deleting between tests as its state cannot be used by another test - all sub directories are now temporary and have been since the wiki was moved to the temporary repo approach.

This delete should get moved in to the prep for integration tests.

Reply:
(I've just done that)